lib/talkable/middleware.rb

```diff
@@ -60,25 +77,44 @@ def sync_uuid_url(uuid)
     end
 
     def sync_uuid_content(uuid)
-      src = CGI::escapeHTML(sync_uuid_url(uuid))
+      src = CGI.escape_html(sync_uuid_url(uuid))
       %Q{
 <img src="#{src}" style="position:absolute; left:-9999px;" alt="" />
       }
     end
 
-    def inject_body?(result)
+    def integration_content
+      integration_script_content + integration_init_content
+    end
+
+    def integration_script_content
+      %Q{
+<script>
+  window._talkableq = window._talkableq || [];
+  _talkableq.push(['init', {
+    site_id: '#{CGI.escape(Talkable.configuration.site_slug)}',
+    server: '#{CGI.escape_html(Talkable.configuration.server)}'
+  }]);
```

Comment:
`push(['init', #{init_parameters.to_json}])` this is the only safe way of doing that.